src/themes/famfamfam/elements/flag.variables

```diff
@@ -16,5 +16,5 @@
 
 @flags: {
 };
-@size-map: {
-};
```

Comment:
Dont remove the semicolon here, to support PHP LESS port

Reply:
I was removed by `no-missing-end-of-source-newline` rule, not sure why, checking

Reply:
I can confirm this was removed by https://github.com/fomantic/Fomantic-UI/pull/2593/commits/c877b76fa1a578d9c4a0f2dd6c9a739c4a6ea3b0 autofix, when the rule is fixed manually, the semicolon stays. Other fixes by `no-missing-end-of-source-newline` are fine (no semicolon removed).